doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -650,8 +662,8 @@ file is a particular case of a module called a *library file*.
    first locating the object file from ``-I`` directives, then
    directly invoking ``Dynlink.loadfile``. For compatibility purposes,
    8.16 still supports this legacy method, with the syntax being
-   ``Declare ML Module "foo_plugin:pkg.plugin.foo".``, where
-   ``foo_plugin`` is the name of the OCaml object file.
+   ``Declare ML Module "pkg_plugin:coq-pkg.plugin.pkg".``, where
+   ``pkg_plugin`` is the name of the OCaml object file.
```

Comment:
Do we want the overloading of `pkg` here? I agree `foo` is not a great name, however note that in general packages have often multiple plugins, such as `coq-core.plugin.ltac` and `coq-core.plugin.tauto` so `coq-pkg.plugin.pkg` seems too restrictive to me.